### DISCUSSION TITLE
i am having problems while registering for student developer pack

### DISCUSSION THREAD

**Discussion by gugabo11** [community/community#63486] [GitHub Education]
hello 
i was trying to register for student developer pack but when it aasked for my id card photo i uploaded one but iit said to take a photo using the take photo icon. i tried that but it said Your application cannot be reviewed until you fix the following:
Please upload proof of your current school affiliation.
the photo has my name class school name in which session was it issued in (2023-24) and my address bcan anyone help

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @gugabo11, we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category and are explicit about what your project entails. I've gone ahead and moved it for you. Good luck!

**Reply by gugabo11** [reply]
thx
